src/__tests__/index.js

```diff
@@ -213,6 +213,41 @@ test('renders a component with theme properties', () => {
   ).toMatchSnapshotWithGlamor()
 })
 
+test('in development mode the theme is frozen and cannot be changed', () => {
```

Comment:
This is technically a breaking change, but manipulation was never truly supported (or useful) so I'm going to let this one slide.